packages/conda-store/src/condaStoreEnvironmentManager.ts

```diff
@@ -28,12 +29,17 @@ interface IParsedEnvironment {
 function parseEnvironment(environment: string): IParsedEnvironment {
   if (environment !== undefined) {
     const [namespaceName, environmentName] = environment.split('/', 2);
+    if (!environmentName) {
+      throw new Error(
+        'Namespace and environment must be provided together, like so: namespace/environment'
```

Comment:
This allows the gator UI to show the user a toast notification containing this error string if the user types environment name, rather than namespace/environment name.